dijet/tasks/jer.py

```diff
@@ -94,10 +93,10 @@ def run(self):
         v_jer = h_jer.view()
 
         # write JER values to output histogram
-        v_jer[index_methods["sm"], :, :].value = jer_sm_val
-        v_jer[index_methods["sm"], :, :].variance = jer_sm_err**2
-        v_jer[index_methods["fe"], :, :].value = jer_fe_val
-        v_jer[index_methods["fe"], :, :].variance = jer_fe_err**2
+        v_jer[index_methods["sm"], :, :].value = np.nan_to_num(jer_sm_val)
+        v_jer[index_methods["sm"], :, :].variance = np.nan_to_num(jer_sm_err**2)
+        v_jer[index_methods["fe"], :, :].value = np.nan_to_num(jer_fe_val)
+        v_jer[index_methods["fe"], :, :].variance = np.nan_to_num(jer_fe_err**2)
```

Comment:
It would be more informative to add the `nan` argument explicitly here.
```suggestion
        v_jer[index_methods["sm"], :, :].value = np.nan_to_num(jer_sm_val, nan=0.0)
        v_jer[index_methods["sm"], :, :].variance = np.nan_to_num(jer_sm_err**2, nan=0.0)
        v_jer[index_methods["fe"], :, :].value = np.nan_to_num(jer_fe_val, nan=0.0)
        v_jer[index_methods["fe"], :, :].variance = np.nan_to_num(jer_fe_err**2, nan=0.0)
```